test/sandbox/app/components/just_joshing_repro_component.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class JustJoshingReproComponent < ViewComponent::Base
+  renders_one :slot1, -> (&block) do
+    # Won't render slot2 content unless I call 'render' here.
+    render JustJoshingReproComponent.new do |c|
+      c.with_slot2(&block)
+    end
```

Comment:
To reviewers:

Callout here. Not sure why I have to call `render` to make this work.

Reply:
Thanks @BlakeWilliams for helping me understand this issue.
I was able to change this code to
```suggestion
    JustJoshingReproComponent.tap do |c|
      c.with_slot2(&block)
    end
```

but I'm still having issues with the test failure.

Reply:
Can you verify if the HTML output looks correct?

It looks like it's failing on line 53, not line 66 as commented (assuming minitest is reporting it correctly)?

> expected to find css "> [data-test-selector='just-joshing-repro-component']" within #<Capybara::Node::Simple tag="div" path="/html/body/div/div"> 1 time but there were no matches